overcast_to_sqlite/html/page.py

```
@@ -26,6 +27,57 @@ def _convert_urls_to_links(text: str) -> str:
+
+
+def _fix_unclosed_html_tags(html_string: str) -> str:
+    """Fix unclosed HTML tags by adding missing closing tags."""
+    if not html_string.strip():
+        return html_string
+        
+    fixer = HTMLTagFixer()
+    try:
+        fixer.feed(html_string)
+        return fixer.get_fixed_html()
+    except Exception:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)